schedule/sle-micro/selinux.yaml

```diff
@@ -0,0 +1,30 @@
+name:           sle_micro_selinux
+description:    >
+    Maintainer: jalausuch@suse.com, qa-c@suse.de.
+    SUSE Linux Enterprise Micro tests
+conditional_schedule:
+  registration:
+    SCC_REGISTER:
+      'installation':
+        - console/suseconnect_scc
+  maintenance:
+    FLAVOR:
+      'MicroOS-Image-Updates':
+        - transactional/install_updates
+schedule:
+  - microos/disk_boot
+  - transactional/disable_grub
+  - '{{registration}}'
+  - '{{maintenance}}'
+  - transactional/enable_selinux
```

Comment:
why is it conditional in main.pm and is without.

Reply:
you are right, this is not needed. The schedule is basically the same as for the image, I will just add the variable. Thanks